calculus.ijs

```diff
@@ -367,47 +374,47 @@ case. <,'&' do.  NB. & - first check for bonded constant
   if. yar opisnoun 0 do.   NB. m&v  (v must be a verb)
     nounarg =. (yar opar 0) vnofaru  NB. m
     verbarg =. (yar opar 1)  NB. v as an AR
-    if. *@#@$ nounarg do. 13!:8 (3) end. NB. except for p. handled above, only atomic m is recognized
+    if. *@#@$ nounarg do. domerr REPORT =: 3;'only atomic m supported in m&v (except for p.) in: ',y, ' ' end. NB. except for p. handled above, only atomic m is recognized
     NB. Look it up in the integral table, execute it on m, return string result
     if. (#primintmandvvb) > primno =. primintmandvvb i. verbarg do.
       (nounarg 1 : (primno {:: primintmandv)) strofu return.
     end.
-    NB. If undifferentiable primitive, fail
-    13!:8 (3)
+    NB. If unintegrable primitive, fail
+    domerr REPORT =: 3;'unintegrable combination: ',y,' '
   elseif. yar opisnoun 1 do.   NB. u&n  (u must be a verb)
     nounarg =. (yar opar 1) vnofaru  NB. n
     verbarg =. (yar opar 0)  NB. u as an AR
-    if. *@#@$ nounarg do. 13!:8 (3) end. NB. except for p. only atomic m is recognized
+    if. *@#@$ nounarg do. domerr REPORT =: 3;'only atomic n supported in u&n in: ',y,' ' end. NB. only atomic m is recognized
     NB. Look it up in the derivative table, execute it on m, return string result
     if. (#primintuandnvb) > primno =. primintuandnvb i. verbarg do.
       (nounarg 1 : (primno {:: primintuandn)) strofu return.
     end.
     NB. If undifferentiable primitive, fail
-    13!:8 (3)
+    domerr REPORT =: 3;'only atomic n supported in u&n: ',y, ' '
   end.
 
   NB. fallthrough cases are u&v
 fcase. ;:'&:@@:' do.  NB. f@g (verb only)
   if. # upoly =. topoly yar opar 0 do.  NB. if u is a polynomial
     if. (-: 1 {.~ -@#) upoly do.
       NB. Handle ^&m for the cases we know
-      if. 1 = #upoly do. ']' return. end.  NB. 1@g, just like 1
+      if. 1 = #upoly do. ']' return. end. NB. 1@g, just like 1
       vop =. yar opstr 1   NB. v as a string
       if. 2 = #upoly do. intstg vop return. end.
       if. (#primintpatopvb) > primno =. primintpatopvb i. <vop do.
         ((<: #upoly) 1 : (primno {:: primintpatopint)) strofu return.
       end.
     end.
-    NB. If undifferentiable primitive, fail
-    13!:8 (3)
+    NB. If unintegrable rimitive, fail
```

Comment:
"rimitive" spelling mistake.

Reply:
Thanks for the suggestions, I fixed this in the next commit.